exercises/isbn-verifier/isbn_verifier.py

```diff
@@ -0,0 +1,10 @@
+def verify(isbn):
+    pass
+
+
+def isbn_generator(isbn):
```

Comment:
`generate_isbn` might be a better function name? I think function names should be more descriptive of what they do than what they are.